samples/parser.ts

```diff
@@ -46,7 +46,7 @@ const getByteLength = (state: Field, buffer: Buffer) => {
 const bufferToReadableData = (
   buffer: Buffer,
   state: Field,
-  nullReplaceTo = '0',
+  nullReplaceTo = '00',
```

Comment:
Possibly not needed to define default value here.

Reply:
It is intended to prevent destructive change.